locations/spiders/sheetz.py

```diff
@@ -1,8 +1,44 @@
+from enum import Enum
+
 import scrapy
 
+from locations.categories import Categories, Extras, Fuel, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
 from locations.items import Feature
 from locations.user_agents import BROWSER_DEFAULT
 
+FEATURES_MAPPING = {
+    "alcohol": None,
+    "atm": Extras.ATM,
+    "bulkDef": None,
+    "carWash": Extras.CAR_WASH,
+    "delivery": None,
+    "driveThru": Extras.DRIVE_THROUGH,
+    "diesel": Fuel.DIESEL,
+    "e0": Fuel.E0,
+    "e15": Fuel.E15,
+    "e85": Fuel.E85,
+    "freeWifi": Extras.WIFI,
+    # TODO: map high flow auto diesel
+    "highFlowAutoDiesel": None,
+    "highFlowDiesel": Fuel.HGV_DIESEL,
+    "kerosene": Fuel.KEROSENE,
+    "open24x7": "opening_hours=24/7",
+    "propane": Fuel.PROPANE,
+    "pncAtm": Extras.ATM,
+    "showers": Extras.SHOWERS,
+    "truckDieselLanes": None,
+    # TODO: availability of truck scales seems important for trucks
+    "truckScales": None,
+    "truckParking": None,
+    "truckParkingSpots": None,
+    "evCcsDcFastCharging": None,
+    "evChaDemoDcFastCharging": None,
+    "evCharger": None,
+    "evTeslaSupercharger": None,
+    "evCharger": None,
```

Comment:
(an item can have multiple amenity values, although there is an argument for yielding an entirely different item for each amenity, and a few spiders use this approach)

Reply:
Strong vote against 1 item 2 categories. It breaks most OSM tooling.

I've mentioned that I think fuel:electric will probably happen in the future before, https://community.openstreetmap.org/t/charging-stations-sites-or-individual-chargers/96810/4

But I want ATP to be closer to OSM, and since it's fuel:electric isn't OSM I probably wouldn't use it here.

Reply:
Let's add it to the TODO list to split items with multiple amenities.

Reply:
I'm parsing charging stations now. Should I split into 2 Items in this spider?

Reply:
Yep I think the idea is to yield 2 items--`amenity/fuel_station` and `amenity/charging_station`. Perhaps convenience stores and restaurant, atms, toilets, etc part of an `amenity/fuel_station` could also potentially be split in the future.

Reply:
What should I do with `ref` in this case?
Also fuel types should not be present for `charging_station`, but shoulb be present for `fuel_station` I guess.

TBH I will be happy to skip `charging_station` for now, add TODO for them and output just `fuel_station`.

Reply:
I'm skipping charging stations in this spider for now. Hopefully I will come back to them later.